sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -522,6 +524,8 @@ case class Range(
   override def computeStats(): Statistics = {
     Statistics(sizeInBytes = LongType.defaultSize * numElements)
   }
+
+  override def outputOrdering: Seq[SortOrder] = output.map(a => SortOrder(a, Descending))
```

Comment:
ordering is the same when `step` in `Range` is positive or negative?

Reply:
Nice catch, thanks! I missed it!